internal/querycoordv2/meta/collection_manager.go

```diff
@@ -126,6 +127,13 @@ func (m *CollectionManager) Recover(broker Broker) error {
 	}
 
 	for _, collection := range collections {
+		// Dropped collection should be deprecated
+		_, err = broker.GetCollectionSchema(context.Background(), collection.GetCollectionID())
```

Comment:
this is a performance bottleneck if many collections loaded, may optimize it by concurrently recover collections

Reply:
ddl requests (including DescribeCollection and ShowPartition) are executed serially in rootcoord